gdk/pixbuf.go

```diff
@@ -513,12 +513,7 @@ func PixbufLoaderNew() (*PixbufLoader, error) {
 	if c == nil {
 		return nil, nilPtrErr
 	}
-
-	obj := &glib.Object{glib.ToGObject(unsafe.Pointer(c))}
-	p := &PixbufLoader{obj}
-	obj.Ref()
-	runtime.SetFinalizer(p, func(_ interface{}) { obj.Unref() })
-	return p, nil
+	return &PixbufLoader{glib.AssumeOwnership(unsafe.Pointer(c))}, nil
```

Comment:
See: newly added documentation for `glib.{Take,AssumeOwnership}` as well as [this comment](https://github.com/gotk3/gotk3/issues/685#issuecomment-751905198).